common-test/pom.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2018-2020 The Feast Authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>feast-parent</artifactId>
+        <groupId>dev.feast</groupId>
+        <version>${revision}</version>
+    </parent>
+
+    <name>Feast Common Test</name>
+    <description>Feast common module with test utilities</description>
+    <artifactId>feast-common-test</artifactId>
```

Comment:
Does something prevent us from using the `common` module for this?

Reply:
yep, in other modules I want to install dependencies of `feast-common-test` only in `test` scope. This is hard to achieve with only `common`

Reply:
So there are two group of dependencies - one gathered in common and one in common-test